README.md

```diff
@@ -3,8 +3,9 @@ This is the Github Project to Annoto GAI work.
 
 ## Initial setup: 
 1. Install required additonal libraries using `requirements.txt`.  
-2. Use the `.env.sample` file in the `config` folder to create an `.env.` file to use with the needed credentials to use UM's OpenAI API.  
-3. For each video, create a folder in the `Captions` Folder, containing the corresponding .srt file within it. The first .srt file found in the subfolder will be used.  
+2. Alongside installing the spaCy NLP library, you will need to download SpaCy's NLP model by running this command: `python -m spacy download en_core_web_sm`
```

Comment:
Why we need to download this library? doesn't `pip install `  meet the requirement for this?

Reply:
No, it would throw an error for me when I tried that. The library appears to be a separate entity from the model from what I understood.

Reply:
So what you meant to say is if you add this `https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.7.1/en_core_web_sm-3.7.1.tar.gz` to `requirments.txt` for equivalent model download it thows error? 

I saw this comment felt might be a better pattern for getting the model. So I suggested the approach.  https://github.com/explosion/spaCy/issues/4577#issuecomment-550159196

Reply:
I feel you should atleast give the exact version to download if the approach I suggested with having it `requirements.txt` doesn't work.

Reply:
I am making a new environment and checking to see if listing the link to the model directly works. 
The spaCy documentation seems to indicate that it does, but it is not working when I use `pip install -r requirements.txt` to try and load the model from GitHub through the link. I am looking at the documentation and FAQs to see how to implement this correctly.

Reply:
I was able to install all the packages and including the URL `.tz.gz` download via `pip install -r requirements.txt` and ran the code. It's working for me.  

Maybe try `pip uninstall -y -r <(pip freeze)` and run the command `pip install -r requirements.txt`.